apps/src/templates/instructions/RubricField.jsx

```diff
@@ -19,7 +20,8 @@ const styles = {
     flexDirection: 'row',
     padding: '4px 10px',
     ':hover': {
-      border: 'solid 1px ${color.light_cyan}',
+      //hover doesn't work when you inject string instead of concat
+      border: 'solid 1px' + color.light_cyan,
```

Comment:
it looks like there is a space missing between '1px' and the color. same below. are you getting the color you want on hover? to eliminate some ambiguity, you can also break these out into 
```js
borderStyle: 'solid',
borderColor: color-light_cyan,
borderWidth: 1
```
that shouldn't be strictly necessary, but might help to debug what's going on here.